app/experimenter/experiments/forms.py

```diff
@@ -860,19 +864,41 @@ def required_reviews(self):
             self["review_bugzilla"],
             self["review_qa"],
             self["review_relman"],
-        )
+        ]
+        return self.add_conditional_reviews(default_required)
+
+    def add_conditional_reviews(self, required_reviews):
+        for review, risk in self._required_reviews_mapping.items():
+            if risk:
+                self._remove_optional_label(review)
+                self._change_help_text_to_risk_help(review)
+                required_reviews.append(self[review])
```

Comment:
I'm always a little hesitant about mutating arguments to a function. In this particular instance we can get away with it because `required_reviews` is newly created, but I'd feel more comfortable doing a shallow copy of `required_reviews` at the beginning of the function.